app/assets/js/rtMedia.js

```diff
@@ -989,7 +989,44 @@ function rtmedia_single_media_alert_message( msg, action ) {
 		action_class = 'rtmedia-warning';
 	}
 
-	jQuery( '.rtmedia-single-media .rtmedia-media' ).css( 'opacity', '0.2' );
+	/**
+	 * Fixed code
```

Comment:
Add a comment about which issue you've fixed instead of just `Fixed code` comment.

Reply:
Comment fixed